dotnet/winforms
# Fixing DoDragDrop AccessViolationException

Fixes #7381

I believe this issue is due to and limited to the new API work done in #6576.

## Proposed changes

- Fix the `AccessViolationException` that occurs when dropping data to `RichTextBox`, and possibly other controls.



## Customer Impact

- Alleviate the serious error.

## Regression? 

- It’s wrong right out of the gate.

## Risk

- Minimal



## Test methodology 

- Write a test to demonstrate the `AccessViolationException`.
- Fix the `AccessViolationException`.
- Verify the fix also fixes the test.
- Run unit and integration tests.

## Test environment(s) 

- 7.0.0-preview.1.22076.8




###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/7528)
